web/src/app/util/SkipToContentLink.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import Link from '@material-ui/core/Link'
+import { makeStyles } from '@material-ui/core'
+
+const useStyles = makeStyles({
+  skipLink: {
+    position: 'absolute',
+    height: '1px',
+    width: '1px',
+    overflow: 'hidden',
+    clip: 'rect(1px, 1px, 1px, 1px)',
+    '&:hover, &:focus, &:active': {
+      top: '5px',
+      left: '5px',
+      zIndex: 100000,
+      clip: 'auto !important',
+      display: 'block',
+      width: 'auto',
+      height: 'auto',
+      padding: '15px 23px 14px',
+      fontWeight: 'bold',
+      fontSize: '14px',
+      textDecoration: 'none',
+      lineHeight: 'normal',
+      color: '#CD1931',
+      backgroundColor: '#ffffff',
+      borderRadius: '3px',
+    },
+  },
+})
+
+export function SkipToContentLink() {
+  const classes = useStyles()
+  return (
+    <Link className={classes.skipLink} href='#content'>
+      Skip to content
```

Comment:
Can you match the MUI typography here? You should be able to wrap `Skip to content` with Typography but retain the `href`.

Also, having it fixed in the top left corner may cause issues with overlap with the Toolbar title. If you render it within the toolbar it should move the Title to the right while it's rendered

Reply:
Let's render it outside the toolbar because of its a header landmark, and we don't want the user to have to navigate into the header landmark to skip repetitive content. I can move it to the right, though.

Reply:
I think either way we will fall into the issue of it covering content, whether it be the title of the page or the actions on the right. Without it being inline with the toolbar I'm not sure where a good place to render it will be. Perhaps covering the GoAlert logo in (or over) the sidebar? Is this only being rendered on desktop?

Reply:
It's mobile as well if you're using a screen reader on mobile it's helpful to jump down to the content.

I understand your concern for covering content. To be able to access the page actions was a keyboard user they would need to tab again to reach the page actions which would hide the skip link.

Reply:
Githubs looks like this

![image](https://user-images.githubusercontent.com/29063970/61227755-88408a80-a6ea-11e9-8ebb-b75418804b8c.png)

Reply:
Targets approach is to push down the bar 
![image](https://user-images.githubusercontent.com/29063970/61227902-dfdef600-a6ea-11e9-8fcb-4fea5d22ab2a.png)

Reply:
I was going to suggest perhaps we push down the bar to make it feel relatively less invasive like how we have on our main website. @arurao @mastercactapus @m17ch @Katy-M what do you think?

Reply:
![image](https://user-images.githubusercontent.com/29063970/61234069-c6907680-a6f7-11e9-8947-ea534e516135.png)

Reply:
I don't feel super strongly, it seems pretty split across the net.

My preference would be the floating one because it doesn't cause content to jump or move when reading/tabbing -- (like GitHub). 

![image](https://user-images.githubusercontent.com/595010/61239102-bc747500-a703-11e9-9868-c9a3a0c631f7.png)
![image](https://user-images.githubusercontent.com/595010/61239124-c72f0a00-a703-11e9-9f04-a7723bdcaabd.png)
![image](https://user-images.githubusercontent.com/595010/61239134-d01fdb80-a703-11e9-9afc-b5be11a96285.png)
![image](https://user-images.githubusercontent.com/595010/61239147-d8781680-a703-11e9-858c-59921251a394.png)
![image](https://user-images.githubusercontent.com/595010/61239247-170dd100-a704-11e9-995e-5620205f45ce.png)

Reply:
@mastercactapus If you don't feel strongly, either way, I think the team wants to go with how Target.com does it. The branch currently reflects this approach and should be ready to merge.

Reply:
Yep, saw that in the chatroom, sounds good!